game/scripts/vscripts/util.ts

```diff
@@ -151,9 +151,12 @@ export function displayDotaErrorMessage(message: string) {
  * Highlights a panel along a path
  * @param path The path along the ui to take, such as "HUDElements/lower_hud/center_with_stats/center_block/inventory"
  * @param duration Optional time in seconds after which to remove the highlight
+ * @param setElementAsParent Optional. Sets the element provided in the path as the parent, instead of as a sibling. Used for cases where the parent has the flow-children CSS property.
  */
-export function highlightUiElement(path: string, duration?: number) {
-    CustomGameEventManager.Send_ServerToAllClients("highlight_element", { path, duration });
+export function highlightUiElement(path: string, duration?: number, setElementAsParent?: boolean) {
+    let eventSetElementAsParent: 0 | 1
+    eventSetElementAsParent = setElementAsParent ? 1 : 0
```

Comment:
remove this after changing the event field to boolean

Reply:
Done.